prow/github/client.go

```diff
@@ -440,6 +440,72 @@ func (c *Client) CreateStatus(org, repo, ref string, s Status) error {
 	return err
 }
 
+func (c *Client) GetRepos(org string, isUser bool) ([]Repo, error) {
+	c.log("GetRepos", org, isUser)
+	var (
+		repos   []Repo
+		nextURL string
+	)
+	if c.fake {
+		repos = append(repos, Repo{Owner: User{Name: "kubernetes"}, Name: "kubernetes", FullName: "kubernetes/kubernetes", HTMLURL: "https://github.com/kubernetes/kubernetes"})
+		repos = append(repos, Repo{Owner: User{Name: "kubernetes"}, Name: "minikube", FullName: "kubernetes/minikube", HTMLURL: "https://github.com/kubernetes/minikube"})
+		repos = append(repos, Repo{Owner: User{Name: "kubernetes"}, Name: "test-infra", FullName: "kubernetes/testinfra", HTMLURL: "https://github.com/kubernetes/test-infra"})
```

Comment:
There should be a better way to structure this besides adding test-related code in the Client methods themselves. Why don't you have a second client implementation that returns these fake repos?

Reply:
Not needed, removed.